r/R/dataset-scan.R

```diff
@@ -42,6 +44,7 @@
 #' - `$UseThreads(threads)`: logical: should the scan use multithreading?
 #' The method's default input is `TRUE`, but you must call the method to enable
 #' multithreading because the scanner default is `FALSE`.
+#' - `$UseAsync(use_async)`: logical: should the async scanner be used?
```

Comment:
I think you have to roxygenize() to get these changes reflected in the actual docs? (and/or you can just `@github-actions autotune` to have CI do that for you)

Reply:
The changes appear to be reflected in `man/Scanner.Rd` now.  Is that sufficient?  Or should I run autotune?

Reply:
I think this is good now, thanks. (I confess I'm not exactly sure what R needs.)